doc/installation/ubuntu.rst

```diff
@@ -6,10 +6,84 @@ Ubuntu Packages
 
 .. index:: ubuntu
 
-There are ready made packages for Ubuntu 14.04 LTS and 16.04 LTS [#ubuntu1604]_.
-These are available in a public ppa repository [#ppa]_,
-so that the installation
-will automatically resolve all dependencies.
+There are ready made packages for Ubuntu.
+Up to privacyIDEA 2.23 packages are available for 
+Ubuntu 14.04 LTS and 16.04 LTS [#ubunut1604]_. 
+These are available via a public ppa repository [#ppa]_.
+
+Starting with privacyIDEA 3.0 packages are available 
+for Ubuntu 16.04LTS and 18.04LTS [#ubuntu1604]_.
+
+Installing privacyIDEA 3.0 or higher
+....................................
+
+Before installing privacyIDEA 3.0 or upgrading to 3.0 you need to add the repository.
+
+.. _add_ubuntu_respository:
+
+Add repository
+~~~~~~~~~~~~~~
+
+Create a new file ``/etc/apt/sources.list.d/privacyidea-community.lst`` with the
+following contents:
+
+   deb http://lancelot.netknights.it/community/xenial/stable xenial main
+
+"xenial" is for Ubuntu 16.04LTS. If you are running Ubuntu 18.04LTS, 
+replase "xenial" with "bionic":
+
+   deb http://lancelot.netknights.it/community/bionic/stable bionic main
+
+Download the signing key:
+
+   wget https://lancelot.netknights.it/NetKnights-Release.asc
+
+Check the fingerprint of the key:
+
+   gpg --with-fingerprint NetKnights-Release.asc
+
+It should look like this:
+
+   pub 4096R/AE250082 2017-05-16 NetKnights GmbH <release@netknights.it>
+   Key fingerprint = 0940 4ABB EDB3 586D EDE4 AD22 00F7 0D62 AE25 0082
+
+Now add the signing key to your system:
+
+   apt-key add NetKnights-Release.asc
+
+
+New installation of privacyIDEA 3.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now run:
+
+   apt update
+   apt install privacyidea-apache2
+
+
+.. _upgrade_ubuntu:
+
+Upgrading privacyIDEA to 3.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to upgrade your privacyIDEA on Ubuntu to privacyIDEA 3.0,
+you need to add a new repository configuration as described in 
+:ref:`_add_ubuntu_respository`.
+
+Now you can simply run:
+
+   apt update
+   apt dist-upgrade
+
+After this it is a good idea to remove old, unused packages by runnint:
```

Comment:
*running

Reply:
done in d69da2fc

Reply:
Oh whoops, seems like the commit didn't fix that typo. I pushed 7032b87 which changes this to "running".